pkgs/applications/science/math/mathematica/versions.nix

```diff
@@ -1,6 +1,22 @@
 { lib, requireFile }:
 
 let versions = [
+  {
+    version = "13.2.0";
+    lang = "en";
+    language = "English";
+    sha256 = "sha256-T9XOXA6jpgN6bcO/do9sw1L73ABtyxuZCLzftv4Cl6o=";
+    installer = "Mathematica_13.2.0_LINUX.sh";
+    webdoc = true;
+  }
+  {
+    version = "13.2.0";
+    lang = "en";
+    language = "English";
+    sha256 = "sha256-YRUvl2H9SwpwDZx04ugd7ZnK5G+t88bzAObXsGGVhk0=";
+    installer = "Mathematica_13.2.0_BNDL_LINUX.sh";
+    webdoc = false;
```

Comment:
Maybe we can parse the `installer` and figure out whether documentation is included or not?

Reply:
Updated.